### ISSUE TITLE
Can't get Rectangle Pro working

### ISSUE THREAD

**Issue by mRGeel** [rxhanson/RectanglePro-Community#252]
Hi,

My windows/apps aren't snapping.
I received the following errors in the log:

2023-04-04T10:19:19+02:00: Failed to find frontmost window.
2023-04-04T10:19:24+02:00: Unable to obtain the accessibility element with the specified attribute at mouse location
![Screenshot 2023-04-04 at 10 21 33](https://user-images.githubusercontent.com/129838432/229732226-1779b637-53cb-4876-ba75-52a526dc326a.png)

The window snap options are greyed out? Any ideas?

**Comment by rxhanson** [comment]
This means that the accessibility privileges provided to Rectangle Pro are not taking effect. Try this:

1. Quit Rectangle Pro 
2. In System Settings > Privacy & Security > Accessibility, disable Rectangle Pro
3. In that same location, remove Rectangle Pro (it is important to do both of those, in that order)
4. Restart your mac, then relaunch & re-enable Rectangle Pro.

**Comment by mRGeel** [comment]
Thanks, this is working now!

Warm regards

Jean Geel
Managing Director and Client Liaison
C  083 278 2189  |  T  012 347 7530 (Please call me on my cellphone)
www.msomiafrica.com<http://www.msomiafrica.com/>


[signature_4090470887]

[A picture containing text, sign  Description automatically generated]<https://www.promoafrica.com/msomiafrica/>

We have moved and are working remotely until further notice
John 3:16<https://www.biblegateway.com/passage/?search=john+3%3A16&version=NET>

From: Ryan Hanson ***@***.***>
Date: Tuesday, 04 April 2023 at 12:40
To: rxhanson/RectanglePro-Community ***@***.***>
Cc: mRGeel ***@***.***>, Author ***@***.***>
Subject: Re: [rxhanson/RectanglePro-Community] Can't get Rectangle Pro working (Issue #252)

This means that the accessibility privileges provided to Rectangle Pro are not taking effect. Try this:

  1.  Quit Rectangle Pro
  2.  In System Settings > Privacy & Security > Accessibility, disable Rectangle Pro
  3.  In that same location, remove Rectangle Pro (it is important to do both of those, in that order)
  4.  Restart your mac, then relaunch & re-enable Rectangle Pro.
